Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -301,40 +346,6 @@ public void StopClimbing()
 		}
 	}
 
-	protected void CheckGroundSlope()
```

Comment:
Did you get this feeling right or are we going with invisible walls on every steep slope? We should mention it to Tasha if so.